Makefile

```
@@ -1,27 +1,31 @@
+PYTHON := $(shell if [ -x .venv/bin/python ]; then printf '%s' .venv/bin/python; else printf '%s' python3; fi)
+
+.PHONY: build test coverage lint format run run-web run-web-only
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)